test/big_natural_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Orangeade.Generator.BigNatural.Tests do
+  use ExUnit.Case, async: true
+  use ExUnitProperties
+  import Orangeade.Generator.BigNatural, only: [stream: 0]
+
+  test "check 5 first generated elements" do
+    generated =
+      stream()
+      |> Caffeine.Stream.take(5)
+
+    assert generated == [
```

Comment:
Did you compare against another generator or check our implementation?

Reply:
No, rather a lazy way. I just assumed our implementation is correct .. what a mistake :/

Reply:
This is a regression test then